src/commands/build.ts

```diff
@@ -74,16 +74,26 @@ export async function debugBuildWithOptions(
 ) {
     const current = ctx.currentFolder;
     if (!current) {
+        console.log("debugBuildWithOptions: No current folder on WorkspaceContext");
```

Comment:
We should be using the output channel to log and not console. Do we not have the no-console lint rule enabled?

Reply:
Done